spot_rl_experiments/spot_rl/utils/skill_executor.py

```diff
@@ -290,6 +296,8 @@ def main():
     rospy.set_param("/skill_name_input", f"{str(time.time())},None,None")
     rospy.set_param("/skill_name_suc_msg", f"{str(time.time())},None,None,None")
     rospy.set_param("/cancel", False)
+    rospy.set_param("place_target_xyz", f"{None},{None},{None}|")
+    rospy.set_param("robot_target_ee_rpy", f"{None},{None},{None}|")
```

Comment:
Maybe we dont need this as line 309 will call these 2 inside the `__init__` for skill executor?